test/parallel/test-tls-alert-handling.js

```diff
@@ -71,15 +70,15 @@ function sendClient() {
 
 
 function sendBADTLSRecord() {
-  var BAD_RECORD = Buffer.from([0xff, 0xff, 0xff, 0xff, 0xff, 0xff]);
-  var socket = net.connect(server.address().port);
-  var client = tls.connect({
+  let BAD_RECORD = Buffer.from([0xff, 0xff, 0xff, 0xff, 0xff, 0xff]);
+  let socket = net.connect(server.address().port);
+  let client = tls.connect({
     socket: socket,
     rejectUnauthorized: false
-  }, function() {
+  }, common.mustCall(function() {
```

Comment:
@paulcamantigue If I'm not mistaken, the place for `common.mustCall()` in this test might be around the callback for the `data` event on `client` (since it will run exactly `max_iter` times on a successful test and no more and no less).

I would recommend removing your current `common.mustCall()` addition and instead adding it around the that event listener along these lines:

```js
client.on('data', common.mustCall(function() {
...
}, max_iter));
...

(I also removed the `chunk` argument to the callback. It is unused as far as I can tell.)